sickrage.py

```diff
@@ -0,0 +1,57 @@
+# Do not edit this script. Edit configuration.py
+import requests
+from datetime import datetime, timezone
+from influxdb import InfluxDBClient
+
+import configuration
+
+current_time = datetime.now(timezone.utc).astimezone().isoformat()
+get_tv_shows = requests.get('{0}/api/{1}/?cmd=future&sort=date&type=missed|today|soon'.format(configuration.sickrage_url, configuration.sickrage_api_key)).json()['data']
```

Comment:
Does Sickrage not allow you to call the commands as headers?

In addition, `{0}{1}` is no different than `{}{}`. I suggest either removing the numbers completely or creating a variable name for meaning e.g. {url}{apikey} (As you can see in mine i prefer the less verbose option)

Reply:
All good! (If you look at my account i helped develop sickrage in the past so i knew the answer just wanted it to spark a thought process for ya haha